tests/core_tests.py

```diff
@@ -531,7 +531,32 @@ def test_user_profile(self):
         data = self.get_json_resp('/superset/fave_dashboards/{}/'.format(userid))
         self.assertNotIn('message', data)
 
+    def test_import_csv(self):
+        self.login(username='admin')
+        config = app.config
+
+        test_file = open('tests/testCSV.csv', 'rb')
```

Comment:
I figured I had to make a file because when I use StringIO to pass in a csv file to `form_data.csv_file`, then `form.csv_file.data = None` during the `POST` request for some reason. I don't know why the value of `form.csv_file.data` always gets deleted unless I pass in a `file` object to `csv_file`. Any ideas on why this happens? If you don't have any ideas, I'll keep trying to make this work over the next couple of days.